README.md

```diff
@@ -36,11 +36,11 @@ You will find the following folders in this repository:
 
 
 ## Contributing with Developer Portal documentation
-We're so glad you're here! Thanks for being interested 
+We're so glad you're here! Thanks for being interested. 
```

Comment:
Opa, acabei não concluindo a frase mesmo. Acho que pode até adicionar mais info tipo "interested in improving our docs"